app/api/routers/breeze_buddy/templates/handlers.py

```
@@ -0,0 +1,163 @@
+            return template
+        else:
+            logger.info(
+                f"No template found for merchant: {merchant_id}, "
+                f"shop_identifier: {shop_identifier}, name: {name}"
+            )
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND,
+                detail=f"Template '{name}' not found for merchant: {merchant_id}"
+            )
+
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)